### PR TITLE
Update RFA for all roles

### PR DESCRIPTION
**Pull request by psykzz**
Depends on #13 
Will update all remaining roles to use the new `notifyRfa`

### CODE REVIEW

**Comment by Neamar** [comment]
Wtf can't review this ;-)

**Comment by psykzz** [comment]
Required the last PR, should be ready now.

**Comment by Neamar** [comment]
Now circle hates you ;-)

**Comment by psykzz** [comment]
Already hates me for windows.

File: game.js [hunk 1]
```diff
@@ -230,50 +251,22 @@ Game.prototype.doctorsAction = function doctorsAction() {
...
+  });
 
-  notifyWerewolves("EAT SOMEONE OM NOM");
+  self.notifyRfa(ROLE_WEREWOLF,
+    {
+      title: "Eat someone",
+      description: "It's time to eat, who do you want to kill?"
+    },
+    players,
+  function(err, choice) {
```
Comment (Neamar) [inline game.js:267]: Choice should be a name, not a player

File: game.js [hunk 2]
```diff
@@ -345,78 +338,38 @@ Game.prototype.setDayAction = function setDayAction() {
...
-          type: "vote",
-          msg: "End of vote, village picked " + victim.name + "."
-        });
-      });
-      self.everyone.forEach(function(p) {
-        p.socket.emit("game event", {
-          type: "killed",
-          killed: [{name: victim.name, role: victim.role}],
-        });
+    self.players.forEach(function(p) {
```
Comment (Neamar) [inline game.js:363]: You probably want to handle the error in the future

Comment (psykzz) [reply game.js:363]: What error here?

File: game.js [hunk 3]
```diff
@@ -345,78 +342,39 @@ Game.prototype.setDayAction = function setDayAction() {
...
-        return;
-      }
+  self.notifyRfa('*',
+    {
+      title: "Hang someone",
+      description: "Decide who of the town shall hang today?"
+    },
+    players,
+  function(err, choice) {
+    var victim = self.getPlayerByName(choice);
```
Comment (Neamar) [inline game.js:361]: That error :)

Comment (psykzz) [reply game.js:361]: but we have no ideal actions if there is an error other than just restart it all again. we need no errors ☠️

**Pull request merged**